src/posts/2021-02-15-phaser-blog.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Phaser'
+comments: true
+author: eline
+date: 2020-02-15
+---
+
+### Wat is Phaser?
+Kort gezegd is [Phaser](https://phaser.io/) een gameframework waarmee je HTML-5 games kan maken. Deze games kunnen gespeeld worden op desktop en mobiel. Phaser kan gratis gedownload worden via https://phaser.io/ en is ontwikkeld door Photon Storm in 2013.
+
+Om gebruik te maken van Phaser is het handig als je al enige kennis hebt van HTML en JavaScript. Phaser is namelijk een JavaScript bibliotheek, wat dus betekent dat je de game volledig hierin bouwt. Phaser maakt het mogelijk voor beginners en gevorderden om een eigen interactieve en leuke game te maken. Verder heeft het veel verschillende functies, zelf te veel om allemaal te benoemen. Dit is ook een van de redenen waarom het zo geliefd is bij game ontwikkelaars. 
```

Comment:
_JavaScript bibliotheek_ -> Ik zou hier persoonlijk JavaScript library van maken. Dit is een algemeen bekende term en ik vind het persoonlijk mooier om de term in het engels te houden. 

_Zelf te veel om allemaal te benoemen_ -> Denk dat dit "zelfs" zou moeten zijn. De zin loopt nu niet lekker